gamemodes/terrortown/gamemode/shared/sh_shopeditor.lua

```diff
@@ -116,7 +137,8 @@ end
 -- @return string name of the equipment
 -- @return ITEM|Weapon equipment table
 -- @realm shared
-function ShopEditor.ReadItemData()
+-- @deprecated
+function ShopEditor.ReadItemDataOld()
```

Comment:
My idea was to change the old shopeditor as less as possible and take as much out of it as possible.
As it gets deleted after the full rework anyway I thought just marking it as deprecated and changing the function would suffice. Because yes this is a breaking change but I should have changed everything that depended on it.

Reply:
I tend to agree with @saibotk. However @ZenBre4ker is right as well. As soon as the shop editor is moved as well, this will be removed anyways. So it isn't that important.

Reply:
Ihr habt recht, war auch nicht schwer... Reine Faulheit es drin zu lassen.
Ist zwar jetzt für die alte Version sehr unnötig, aber dafür clean.